Makefile

```diff
@@ -50,8 +50,8 @@ mindebug:
 		api web-dev rpc-default
 	@echo 'Waiting a moment before running migrations'
 	sleep 3
-	@echo 'running migrations'
```

Comment:
This is going to break the [other pr](https://github.com/GusSand/Anubis/pull/137). When we merge, you should rebase master, and wrangle the conflict in this branch.

Reply:
Oh sorry, I forgot to delete this.

Reply:
Alright, I changed it back to what it was.